server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -0,0 +1,273 @@
+import os
+import logging
+import pytest
+
+from pathlib import Path
+from typing import List
+
+from pbench.process_tb import ProcessTb, Results
+from pbench.test import tarball, bad_tarball, fixture_path
+
+
+class MockConfig:
+    TS = "run-1970-01-02T00:00:00.000000"
+
+    def get(self, section: str, option: str) -> str:
+        return ""
```

Comment:
If you use the following definition for `MockConfig`,
```
class MockConfig:
    TS = "run-1970-01-02T00:00:00.000000"

    def __init__(self, mappings: dict):
        self.mappings = mappings

    def get(self, section: str, option: str) -> str:
        assert (
            section in self.mappings and option in self.mappings[section]
        ), f"Unexpected configuration option, {section!r}/{option!r}, in MockConfig.get()"
        return self.mappings[section][option]

```
then you don't need to patch the `get()` method with a separate mock in each test.  You simply instantiate a `MockConfig` object specifying the config mappings to the c'tor, and then the `get()` method will work "as is":
```
        mappings = {
            "pbench-server": {
                "pbench-receive-dir-prefix": tb_path,
                "put-token": "Authorization-token",
            }
        }
        ptb = ProcessTb(MockConfig(mappings), test_logger)
```
I think the result is cleaner and simpler, and it's slightly shorter in each test function.